src/electron-forge-start.js

```diff
@@ -28,15 +28,19 @@ const main = async () => {
 
   dir = await resolveDir(dir);
   if (!dir) {
-    startSpinner.fail();
+    locateSpinner.fail();
     console.error('Failed to locate startable Electron application'.red);
     if (global._resolveError) global._resolveError();
     process.exit(1);
   }
+  locateSpinner.succeed();
```

Comment:
I think that's a refactor for another PR.  The `locate` spinner / error messages can probably be reused across most of the commands.